<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> Файл</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Replace"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Формат</string>
    </property>
    <addaction name="action_Bold"/>
    <addaction name="action_Italic"/>
    <addaction name="action_Underline"/>
    <addaction name="separator"/>
    <addaction name="action_Left"/>
    <addaction name="action_Center"/>
    <addaction name="action_Right"/>
    <addaction name="action_Justify"/>
    <addaction name="separator"/>
    <addaction name="action_Color"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Стили</string>
     </property>
     <addaction name="actionWindows"/>
     <addaction name="actionFusion"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Языки</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Bold"/>
   <addaction name="action_Italic"/>
   <addaction name="action_Underline"/>
   <addaction name="separator"/>
   <addaction name="action_Left"/>
   <addaction name="action_Center"/>
   <addaction name="action_Right"/>
   <addaction name="action_Justify"/>
   <addaction name="separator"/>
   <addaction name="action_Color"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/new.png</normaloff>:/icon/assets/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/open.png</normaloff>:/icon/assets/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/save.png</normaloff>:/icon/assets/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/save.png</normaloff>:/icon/assets/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/editundo.png</normaloff>:/icon/assets/icons/textedit/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/editredo.png</normaloff>:/icon/assets/icons/textedit/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/cut.png</normaloff>:/icon/assets/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/copy.png</normaloff>:/icon/assets/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/paste.png</normaloff>:/icon/assets/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>Найти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>Заменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textbold.png</normaloff>:/icon/assets/icons/textedit/textbold.png</iconset>
   </property>
   <property name="text">
    <string>Жирный</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textitalic.png</normaloff>:/icon/assets/icons/textedit/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>Курсив</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textunder.png</normaloff>:/icon/assets/icons/textedit/textunder.png</iconset>
   </property>
   <property name="text">
    <string>Подчеркнутый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textleft.png</normaloff>:/icon/assets/icons/textedit/textleft.png</iconset>
   </property>
   <property name="text">
    <string>Влево</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="action_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textcenter.png</normaloff>:/icon/assets/icons/textedit/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>По центру</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textright.png</normaloff>:/icon/assets/icons/textedit/textright.png</iconset>
   </property>
   <property name="text">
    <string>Вправо</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="action_Justify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Reader.qrc">
     <normaloff>:/icon/assets/icons/textedit/textjustify.png</normaloff>:/icon/assets/icons/textedit/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>По ширине</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+J</string>
   </property>
  </action>
  <action name="action_Color">
   <property name="text">
    <string>Цвет</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Предварительный просмотр</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Немецкий</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Reader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Undo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Redo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
